libs/partners/zenguard/pyproject.toml

```diff
@@ -0,0 +1,85 @@
+[tool.poetry]
+name = "langchain-zenguard"
+version = "0.1.0"
+description = "An integration package connecting Zenguard and LangChain"
+authors = []
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain"
+license = "MIT"
+
+[tool.poetry.urls]
+"Source Code" = "https://github.com/langchain-ai/langchain/tree/master/libs/partners/zenguard"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+langchain-core = "^0.2.0"
+
```

Comment:
do we need zenguard dependency?

Reply:
due to the fact that it uses pydantica v1, I could not initialize the Zenguard client in the constructor and decided to contact the API completely manually